drivers/SmartThings/zigbee-window-treatment/src/aqara/roller-shade/init.lua

```diff
@@ -33,6 +33,8 @@ local function window_shade_open_cmd(driver, device, command)
   local initialized = device:get_latest_state("main", initializedStateWithGuide.ID,
     initializedStateWithGuide.initializedStateWithGuide.NAME) or 0
   if initialized == initializedStateWithGuide.initializedStateWithGuide.initialized.NAME then
+    aqara_utils.disable_update_shade_level(device)
+    device:emit_event(capabilities.windowShadeLevel.shadeLevel(100))

```

Comment:
There's no guarantee that the shade will actually go to 100, right? What if there are transmission errors?

Reply:
Okay. I'll fix it.